MotorState[<=16] motor_states
